src/assets/news/2024-06-07 - Lemmy Release v0.19.4 - Image Proxying and Federation improvements.md

```diff
@@ -0,0 +1,119 @@
+- [Indicate to user when they are banned from community](https://github.com/LemmyNet/lemmy/pull/4458)
+- [Added alt_text for image posts](https://github.com/LemmyNet/lemmy/pull/4477)
+- [Dont require leading ! or @ to fetch a user or community](https://github.com/LemmyNet/lemmy/pull/4513)
+- [Extra fields for PostReport and CommentReport views](https://github.com/LemmyNet/lemmy/pull/4520)
+
+### Full Changelog
+
+- [API changes - lemmy-js-client 0.19.3 -> 0.19.4](https://github.com/LemmyNet/lemmy-js-client/compare/0.19.3-alpha.1...0.19.4)
+- [Lemmy Backend](https://github.com/LemmyNet/lemmy/pulls?q=is%3Apr+merged%3A%3E2024-01-22)
+- [Lemmy-UI](https://github.com/LemmyNet/lemmy-ui/pulls?q=is%3Apr+merged%3A%3E2024-01-22)
+
+## Upgrade instructions
+
+**Warning: This version requires both a Postgres and Pictrs version upgrade, which requires manual intervention.**
+
+Follow the upgrade instructions for [ansible](https://github.com/LemmyNet/lemmy-ansible/blob/main/UPGRADING.md) or [docker](https://join-lemmy.org/docs/en/administration/install_docker.html#updating).
+
+If you need help with the upgrade, you can ask in our [support forum](https://lemmy.ml/c/lemmy_support) or on the [Matrix Chat](https://matrix.to/#/!OwmdVYiZSXrXbtCNLw:matrix.org).
+
+## Thanks to everyone
+
+We'd like to thank our many contributors and users of Lemmy for coding, translating, testing, and helping find and fix bugs. We're glad many people find it useful and enjoyable enough to contribute.
+
+Special thanks goes to [Radically Open Security](https://www.radicallyopensecurity.com/), @sleepless and @matc-pub for their work on lemmy-ui and lemmy-ui-leptos, @dullbananas for their help cleaning up the back-end, DB, and reviewing PRs, @phiresky for federation work, @MV-GH for their work on Jerboa and API suggestions, @asonix for developing pictrs, @ticoombs and @codyro for helping maintain lemmy-ansible, @kroese, @povoq, @flamingo-cant-draw, @aeharding, @Nothing4U, @db0, @MrKaplan, for helping with issues and troubleshooting, and too many more to count.
+
+## Support development
+
+We (@dessalines and @nutomic) have been working full-time on Lemmy for over three years. This is largely thanks to support from [NLnet foundation](https://nlnet.nl/), as well as [donations from individual users](https://join-lemmy.org/donate).
+
+If you like using Lemmy, and want to make sure that we will always be available to work full time building it, consider [donating to support its development](https://join-lemmy.org/donate). A recurring donation is the best way to ensure that open-source software like Lemmy can stay independent and alive, and helps us grow our little developer co-op to support more full-time developers.
+
+- [Liberapay](https://liberapay.com/Lemmy) (preferred option)
+- [Open Collective](https://opencollective.com/lemmy)
+- [Patreon](https://www.patreon.com/dessalines)
+- [Cryptocurrency](https://join-lemmy.org/donate) (scroll to bottom of page)
```

Comment:
Is there an anchor on that page we could link to for future dev updated and release notes?

Reply:
I forgot, we added this page: https://join-lemmy.org/crypto

I'll link that instead.